### ISSUE TITLE
unipept pept2lca --select bug with fasta headers

### ISSUE THREAD

**Issue by Autom3** [unipept/unipept-cli#1]
When I tried running the command `cat test | unipept pept2lca --select taxon_id` I got `fasta_header,taxon_id` as output.

Contents of the test file:
```
>MG00HS19:771:HKNTHBCXX:2:1101:8323:3895_1_317_+
VGFLVQGTLYPDVIESASPK
GGPSVTIK
THHNVGGLPEQLPFK
LIEPLR
ELGLPEEMVGR
HPFPGPGLAIR
ILGEVTQDQLDILR
ADHIYTE
>MG00HS19:771:HKNTHBCXX:2:1101:8643:3889_1_129_-
EQDGGTSR
ALAESEGTTK
QAQQEAAR
IAFER
LLLEESQEGS
```

This seems to only happen when I add fasta headers to the file.

**Comment by bmesuere** [comment]
Thanks for the report, I'll take a look at it. 
Workaround: also select the peptide field as output (and use `cut` to remove it if needed)
Reason why this happens: The response from our API server is mapped back to the input peptides before generating output. This is done to preserve the order of the input. Because "peptides" isn't in the output, notting gets mapped and the output is empty.

**Comment by bmesuere** [comment]
This was fixed in version 1.1.1. Sorry for the wait!
